torch/nn/modules/linear.py

```diff
@@ -224,5 +229,6 @@ def initialize_parameters(self, input) -> None:  # type: ignore
             with torch.no_grad():
                 self.in_features = input.shape[-1]
                 self.weight.materialize((self.out_features, self.in_features))  # type: ignore
+                self.bias.materialize((self.out_features,))  # type: ignore
```

Comment:
Shouldn't there be a check similar to conv here for the case where the users gave bias=False?

Reply:
Thank you for the catch! I've updated the  code.